### ISSUE TITLE
Page effects not working

### ISSUE THREAD

**Issue by lmvco** [readium/readium-js-viewer#301]
I am doing tests to release 0.17 and the page effects (slide, fade, etc) are not working as expected.

Regardless of value that i choose, on page turn, no effect is applied. Is this a known issue? There's any work around?

I tested also in app available in Chrome Store - https://chrome.google.com/webstore/detail/readium/fepbnnnkkadjhjahcafoaglimekefifl - and it's not working also.

**Comment by rkwright** [comment]
You are correct.  They don't seem to work.  I believe they should be removed until we have time to make them work properly.  Thanks for reminding me of this.

**Comment by danielweck** [comment]
I agree with Ric, but note that if page transition effects do not work in fixed layout ebooks, then it is indeed a bug (is that the case?). However if they do not work in reflowable Epubs, then it is the expected behaviour (feature not implemented yet).

**Comment by lmvco** [comment]
Yes, I confirm that it is working fine for fixed layout books. Thanks for clarification.

Is there any estimated date for same feature in reflowable books?

Thanks

**Comment by danielweck** [comment]
No ETA I'm afraid, the (small team of) developers are currently focusing on higher-priority tasks ... thus why Ric suggests we drop / hide the feature altogether, to avoid confusing end-users with the reflowable / pre-paginated discrepancy). Would you be able to contribute code yourself?

**Comment by lmvco** [comment]
I am very busy now but I will try to contribute anyway. Can you share the details of current implementation for fixed layout books? Which is the module that implement this?
Based on that, I will investigate how to implement effects in reflowables...

**Comment by danielweck** [comment]
https://github.com/readium/readium-shared-js/blob/master/js/views/one_page_view.js#L56

Look for `pageSwitchDir` to see how the direction of the visual transition is handled, and as you can see it's a simple array of hard-coded possible choices:

```javascript
        var _pageTransitions = [];
        _pageTransitions.push(_pageTransition_OPACITY); // 0
        _pageTransitions.push(_pageTransition_TRANSLATE); // 1
        _pageTransitions.push(_pageTransition_ROTATE); // 2
        _pageTransitions.push(_pageTransition_SWING); // 3
```

PS: we're working on a plugin architecture at the moment, "page transitions" would be an excellent candidate for refactoring into an extension point, and a default plugin implementation.

**Comment by rkwright** [comment]
Just confirmed that it DOES work in fixed-layout books  But until it works in both fixed and reflowable, I think we should remove it.  We can put it back when we have a good solution that isn't confusing to end-users.
